frontend/src/components/transactions/ListOfTransactions.js

```diff
@@ -1,55 +1,52 @@
 import React from "react";
-
 class Transactions extends React.Component {
   state = {
         transactionList : [{
-                                 id: 1,
-                                 firstname: 'Robin',
-                                 lastname: 'Wieruch',
-                                 amount: 1988,
-                                 date: '10/12/2018',
+                                 ID: 1,
+                                 Name: 'Robin',
+
+                                 Description: 'Samsung mobile',
+                                 Status: 'In stock',
                                },
                                {
-                                 id: 2,
-                                 firstname: 'Dave',
-                                 lastname: 'Davidds',
-                                 amount: 1990,
```

Comment:
Check with Tejshree about amount field in database!

Reply:
The field is there but we have not implemented the functionality.